### DISCUSSION TITLE
SSO (Google authenticator)

### DISCUSSION THREAD

**Discussion by Touchmeshuu11** [gravitational/teleport#6758] [Q&A] [answered]
can anyone tell me, how many user can i create with google authenticator(SSO) on teleport free version ?

**Comment by tooboredtocode** [comment, marked as answer]
As far as I know google authenticator is not an sso service but an app generating 2fa codes. Those are used by local users and I don't know how many of those you can create. (probably more than you'll ever need)

Anyhow, if you mean Google Identity (Googles Sign In using OIDC) that is limited to Teleport Enterprise and not available on the OSS Edition.

**Reply by Touchmeshuu11** [reply]
ok, thank's
